eve/workers/pod.yaml

```diff
@@ -73,7 +73,7 @@ spec:
         - -ec
         - |
           # wait for exports to be created
-          sleep 10
+          sleep 15 
```

Comment:
So I tried a `if [[ ! -e /root/.aws/exports ]]` and `while [[ ! -e /root/.aws/exports ]]` but had the same issue as before where it would never find the files (but flaky as sometimes it would find it).  I reverted back to the sleep command for now until there's a better way to apply secrets. Do you still approve @jonathan-gramain ?

https://eve.devsca.com/github/scality/s3/#/builders/3/builds/3035
https://eve.devsca.com/github/scality/s3/#/builders/3/builds/3040